src/Runner.Worker/Handlers/ScriptHandler.cs

```diff
@@ -24,16 +24,22 @@ public sealed class ScriptHandler : Handler, IScriptHandler
 
         protected override void PrintActionDetails(ActionRunStage stage)
         {
-
-            if (stage == ActionRunStage.Post)
```

Comment:
Intentionally removing this, we shouldn't have a way to access this as a script step anyways